staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -48,7 +48,8 @@ const (
 	aesGCMTransformerPrefixV1    = "k8s:enc:aesgcm:v1:"
 	secretboxTransformerPrefixV1 = "k8s:enc:secretbox:v1:"
 	kmsTransformerPrefixV1       = "k8s:enc:kms:v1:"
-	kmsPluginHealthzTTL          = 3 * time.Second
+	kmsPluginHealthzNegativeTTL  = 3 * time.Second
+	kmsPluginHealthzPositiveTTL  = 90 * time.Second
 )
```

Comment:
I think this is still too much for the integration test to pass.
I believe this would need to be under 30 seconds, so I suggest 20.
We can follow up with a PR to expose cache TTL so that the integration test could run in reasonable amount of time.

Reply:
done